src/pages/Main.tsx

```diff
@@ -83,7 +82,7 @@ export const Main = () => {
       });
     }
 
-    feedItems = orderBy(feedItems, ["created"], ["desc"]);
+    feedItems.sort((a, b) => (a.created > b.created) ? -1 : ((b.created > a.created) ? 1 : 0));
```

Comment:
Could you create a separate function so that it's clearer what it does?

Reply:
and cover it with tests for higher code quality

Reply:
It's a onetime function using a native sort. I don't think it needs a layer of abstraction as we'd just reinvent `lodash`. We should aim to keep it simple.

Reply:
Re. tests, it is commented out in CI/CD and doesn't currently work and I consider that to be outside the scope of this ticket. It should be fixed, but here.

Reply:
I agree with the tests.

Reply:
However, for better DX and to reduce the mental load when reading the code, it's better to create a separate function with a clear name.